wagtail/core/management/commands/rm_cropped_images.py

```diff
@@ -0,0 +1,53 @@
+import os, shutil
+from django.conf import settings
+from django.apps import apps
+from django.core.management.base import BaseCommand, CommandError
+
+# we check if there is a custom image model and rendition model
+# if a custom image model exist we introduce a extra setting
+if hasattr(settings, 'WAGTAILIMAGES_IMAGE_MODEL_RENDITION'):
+    app, model = settings.WAGTAILIMAGES_IMAGE_MODEL_RENDITION.split('.')
```

Comment:
Can you get the Rendition model using its foreign key to a custom image model instead? This extra setting for one small use case seems redundant. If it is necessary, it should be documented in the settings reference.

Reply:
I tried, and this is annoying but I didn't see an good way to link the two models and detact what the model of the renditions would be? Do you have any suggestions so we can drop that extra Wagtail setting.

Reply:
When you've retrieved the image model, I think you should be able to use its classmethod `get_rendition_model`

Reply:
ah if so I'll look into it this weekend then can drop the extra setting :)

Reply:
Great, thank you!